waku/v2/protocol/pb/waku_message.proto

```diff
@@ -2,6 +2,16 @@ syntax = "proto3";
 
 package pb;
 
+message WakuMessage {
```

Comment:
Shouldn't this also come from https://github.com/waku-org/waku-proto/blob/main/waku/message/v1/message.proto to avoid duplication?

Also this seems to be different from the one in proto repo.

Reply:
Forgot to commit this